common/libs/framing/soft_rx_buffer.h

```diff
@@ -0,0 +1,46 @@
+#ifndef __SOFT_RX_BUFFER_H
+#define __SOFT_RX_BUFFER_H
+#include "serial_listeners.h"
+#include <stdint.h>
+
+template <int RX_BYTES_MAX> class SoftRxBuffer {
```

Comment:
Is this class only for testing? If yes, please say so and maybe even #ifdef it on TEST_MODE

Reply:
No, it's also used for rx from the GUI side